app/models/katalyst/content/item.rb

```diff
@@ -43,6 +43,10 @@ def initialize_tree
         self.parent   ||= nil
         self.children ||= []
       end
+
+      def validate_background?
+        true
+      end
```

Comment:
LEGTM

alternative would be to have another background option 'inherit' or something which we defaulted to